modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -62,12 +62,19 @@ scp your_username@remotehost:/opt/anypoint/runtimefabric/backup.tar.gz /backup-p
 
 == Restore a Cluster
 
+[WARNING]
+====
+Restore must be performed to the same Runtime Fabric Cluster in Control Plane (Same Runtime Fabric cluster in Runtime Manager)
```

Comment:
This sentence confuses me a little. Is it:
You must restore the Runtime Fabric Cluster to the same control plane or Runtime Manager in which it originated.